src/types/redis_bitmap.cc

```diff
@@ -226,14 +226,20 @@ rocksdb::Status Bitmap::BitCount(const Slice &user_key, int64_t start, int64_t s
 
   if (metadata.Type() == kRedisString) {
     redis::BitmapString bitmap_string_db(storage_, namespace_);
-    return bitmap_string_db.BitCount(raw_value, start, stop, cnt);
+    return bitmap_string_db.BitCount(raw_value, start, stop, is_bit, cnt);
   }
 
+  auto totlen = static_cast<int64_t>(metadata.size);
+  if (is_bit) totlen <<= 3;
   // Counting bits in byte [start, stop].
-  std::tie(start, stop) = BitmapString::NormalizeRange(start, stop, static_cast<int64_t>(metadata.size));
+  std::tie(start, stop) = BitmapString::NormalizeRange(start, stop, totlen);
   // Always return 0 if start is greater than stop after normalization.
   if (start > stop) return rocksdb::Status::OK();
 
+  uint8_t first_byte_neg_mask = 0, last_byte_neg_mask = 0;
+  std::tie(start, stop) =
```

Comment:
It seems like from this line, `start` and `end` means change from start BIT/BYTE to start BYTE. If it's true, can you change the params name? 
Variables with different meanings require different names.